src/object.d

```diff
@@ -2807,20 +2824,37 @@ auto byKey(T : V[K], K, V)(T* aa) pure nothrow @nogc
 ///
 @safe unittest
 {
-    auto dict = [1: 0, 2: 0];
-    int sum;
+    auto dict = [1: "v1", 2: "v2"];
+    int sum = 0;
```

Comment:
Unclear why this change is made.

Reply:
In the other examples, the keys and values are of different types and it's the values being added up. When I looked at this example, it honestly took me a bit of time to figure out that it's the keys being added up, so I thought having different types for keys and values might be an improvement.

About the initialization of `sum`: I was under the impression that using uninitialized variables was a bad practice, even in D land. I was wrong.